mgmtsystem_action/__openerp__.py

```diff
@@ -20,31 +20,25 @@
 ##############################################################################
 {
     "name": "Management System - Action",
-    "version": "8.0.1.2.0",
+    "version": "9.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Management System",
-    "description": """\
-This module enables you to manage the different actions of your management
-system:
-  * immediate actions
-  * corrective actions
-  * preventive actions
-  * improvement opportunities.
-""",
-    "depends": ['mgmtsystem', 'crm_claim'],
+    "depends": ['mgmtsystem', 'crm_claim', 'mail'],
```

Comment:
Hi Daniel. The crm_claim module is important. It provide some fields such as date_closed. Except opening_date that we create, the order date are coming from crm_clain and we use them to develop pivot view.